client/Components/Pages/RecipePage.jsx

```diff
@@ -70,13 +76,42 @@ class RecipePage extends Component {
     })
   }
 
+  backClick() {
+    this.setState({
+      displayRecipe: true,
+      addRecipe: false
+    })
+  }
+
+  onFocus(event) {
+    this.setState({
+      imageErrorStatus: false,
+      imageError: ''
+    });
+  }
+
   onSubmit(event) {
     event.preventDefault();
-    if(this.state.imageHeight < 200 || this.state.imageWidth < 200) {
+    if(this.state.recipeName.trim() === '' 
+    || this.state.details.trim() === '' 
+    || this.state.ingredient.trim() === '') {
+      toastr.options = {
+        "debug": false,
+        "timeOut": "2000",
+        "showEasing": "swing",
+        "hideEasing": "linear",
+        "showMethod": "fadeIn",
+        "hideMethod": "fadeOut"
+      };
+      toastr.error('All field are required')
+    }
+    else if(this.state.imageHeight < 200 
+      || this.state.imageWidth < 200 
+      || !this.state.image) {
       this.setState({
         imageErrorStatus: true,
         loading: false,
-        imageError: 'Image is too small'
+        imageError: 'Image is too small or no Image provided'
```

Comment:
Can you make this error message clear? If you have two cases for when the error is to be displayed, how about having a condition that checks if it is too small or no image is being provided.

Reply:
Will work on that, thanks